src/main/java/io/fabric8/maven/docker/model/InspectedContainer.java

```diff
@@ -0,0 +1,31 @@
+package io.fabric8.maven.docker.model;
+/*
+ *
+ * Copyright 2014 Roland Huss
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Interface representing an inspected container
+ *
+ * @author daniel
+ * @since 17/02/15
+ */
+public interface InspectedContainer extends Container {
```

Comment:
`ContainersListElement` can not provide these properties since they are only available on the docker inspect endpoint (single container) and not the list container endpoint. To be honest, right now i can not even find the healthy field documented  it in the official docker api docs, I took the path directly from the docker source. I'll try to provide the original reference asap.